data/backstage/infra/2025-10-30-how-the-opam-repository-works.md

```diff
@@ -0,0 +1,105 @@
+
+The repository maintains quality through several key policies documented in the [repository wiki](https://github.com/ocaml/opam-repository/wiki):
+
+**Package Utility**: Packages must provide substantial utility to the OCaml community. Single-function modules or packages with minimal functionality are generally rejected to preserve namespace and repository efficiency.
+
+**Naming and Security**: Package names are carefully reviewed to prevent confusion with existing packages and potential security risks. Name squatting and packages with deceptively similar names to established libraries are not permitted.
+
+**Dependency Constraints**: The repository discourages overly strict version constraints (using `=` or `<=`) as they make package combination difficult and worsen the user experience. Instead, packages should use reasonable upper bounds and allow opam's solver to find compatible combinations.
+
+**Maintenance Requirements**: Packages must be actively maintained for modern OCaml versions. The current compiler cutoff threshold is OCaml 4.08, meaning packages that don't support recent compiler versions may be archived.
+
+### The Archiving Process
+
+To manage repository growth and maintain quality, the opam repository has implemented a systematic archiving process for unmaintained packages. Packages that no longer meet the compiler cutoff threshold or are explicitly marked as unmaintained get moved to the [opam-repository-archive](https://github.com/ocaml/opam-repository-archive).
+
+The process works through maintenance intent declarations using the `x-maintenance-intent` field in package metadata. When maintainers stop supporting older versions, the community is notified, and if no new maintainer steps forward, packages are archived rather than removed entirely.
+
+## The Human Element
+
+### Current Maintainer Team
+
+The repository is maintained by a dedicated team of volunteers who review every package submission. After Kate Deplaix (kit-ty-kate) retired from opam-repository maintenance in 2024 following six and a half years of service to focus on opam itself, the current active maintainers include @mseri and @raphael-proust who took over primary responsibilities. Kate's contributions were instrumental in maintaining the repository's quality and establishing many of the current processes and documentation.
+
+### Community Involvement
+
+The maintainer team actively seeks new volunteers to help manage the growing submission volume. The role offers excellent opportunities to:
+
+- Connect with and support contributors from across the OCaml ecosystem
+- Learn advanced packaging techniques and build system intricacies  
+- Contribute to ecosystem sustainability and health
+- Gain experience with large-scale open source project management
+
+New maintainers start with triaging access to review and approve changes, and after demonstrating success, can receive full access. The process includes orientation sessions for new volunteers.
```

Comment:
Can we leave this out? It may change and so I suggest instead just linking to https://github.com/ocaml/opam-repository/wiki/Onboarding-documentation